src/modules/AutoinstStorage.rb

```diff
@@ -1125,12 +1126,18 @@ def Write
       changed = false
       Builtins.foreach(@AutoTargetMap) do |device, data|
         if !tm.has_key?(device) && data.fetch("type",:CT_DISK)==:CT_DISK
-          Report.Error(
-            Builtins.sformat(
-              _("device '%1' not found by storage backend"),
-              device
+          if device && !device.empty?
+            Report.Error(
+              Builtins.sformat(
+                _("device '%1' not found by storage backend"),
+                device
+              )
             )
-          )
+          else
+            Report.Error(
+              Builtins.sformat(_("No device found by storage backend"))
```

Comment:
np: I would said "No suitable device was found...". It is not that the device does not exist, but that the device is skipped. What do you think?